django/santropolFeast/member/models.py

```diff
@@ -158,30 +158,30 @@ class Client(models.Model):
     DECEASED = 'I'
 
     CLIENT_STATUS = (
-        (PENDING, _('pending')),
-        (ACTIVE, _('active')),
-        (PAUSED, _('paused')),
-        (STOPNOCONTACT, _('stopnocontact')),
-        (STOPCONTACT, _('stopcontact')),
-        (DECEASED, _('deceased')),
+        (PENDING, _('Pending')),
+        (ACTIVE, _('Active')),
+        (PAUSED, _('Paused')),
+        (STOPNOCONTACT, _('Stop: no contact')),
+        (STOPCONTACT, _('Stop contact')),
```

Comment:
We are no business logic about that for the moment (i think).
Personnaly, i like the ```:```